lib/taskana-core-test/src/test/java/acceptance/task/create/CreateTaskAccTest.java

```diff
@@ -332,21 +333,24 @@ void should_CreateTask_When_MultipleAttachmentsAreSpecified() throws Exception {
     Instant earlierInstant = Instant.parse("2018-01-12T00:00:00Z");
     Instant laterInstant = Instant.parse("2018-01-15T00:00:00Z");
     Task task = createDefaultTask();
+    Instant now = Instant.now();
+    Instant roundedNow = now.truncatedTo(ChronoUnit.MILLIS);
+
     Attachment attachment =
         TaskAttachmentBuilder.newAttachment()
             .objectReference(defaultObjectReference)
             .classificationSummary(defaultClassificationSummary)
-            .created(Instant.now())
-            .modified(Instant.now())
+            .created(roundedNow)
+            .modified(roundedNow)
             .received(laterInstant)
             .channel("E-MAIL")
             .build();
     Attachment attachment2 =
         TaskAttachmentBuilder.newAttachment()
             .objectReference(defaultObjectReference)
             .classificationSummary(defaultClassificationSummary)
-            .created(Instant.now())
-            .modified(Instant.now())
+            .created(roundedNow)
+            .modified(roundedNow)
```

Comment:
created and modified should be set during creation of attachment (during Task creation). Can you try to remove setting created and modified? They should be set automatically. If the test stays flaky, we will need to round the created and modified before comparing them.